app/posts/manage-teacher-training-applications/2021-04-08-gathering-school-direct-contact-details-from-higher-education-institutions.md

```diff
@@ -0,0 +1,123 @@
+
+After testing that design we decided to explore a simpler approach, focusing on gathering contact details for SDs.
+
+## What we did and why
+
+We created a spreadsheet template that we could send to HEIs as a CSV file. The columns in the template were:
+
+- partner (pre-filled with the name of an SD) 
+- code (pre-filled with the SD’s provider code)
+- contact name (to be completed by the HEI user)
+- contact email (to be completed by the HEI user)
+
+We decided to keep the headings short as we have no control over column width in a CSV file. When we tested it using Microsoft Excel, the default column width was very narrow.
+
+If an SD was already using Manage, the contact name and email cells would be pre-filled with ‘Not needed - already has account’. 
+
+We also created an email explaining what we wanted users to do. It asked them to either: 
+
+- download the spreadsheet, complete it and email it back
+- fill in a Google Sheets version of the spreadsheet online
+
+The email said that courses would become available as soon as the contact at the SD set up their account.
+
+Since the previous iteration, we’d discovered that HEI users have to sign a data sharing agreement (DSA) in order to get access to the ‘sandbox’ version of Manage. They access this so that they can experiment before signing up to use the service with real data.
+
+Since each HEI should have already signed a DSA, we decided not to ask them to do it as part of our process.
+
+## Research findings
+
+Users understood what they were being asked to do in the email.
+
+Some users said they would not want to complete the spreadsheet immediately. They’d want to make sure they were ready to use Manage and also let SDs know what was happening.
+
+When we asked users to complete the spreadsheet, we found that in general they: 
+
+- expected to invite all their SDs at the same time
+- did not recognise the provider code we used – some expected the unique reference number (URN) instead
+- wanted to see all their SDs listed, including the ones which were already set up in Manage 
+- would add any missing information, such as SDs which should have been on the list 
+- were unsure about what to do if something was incorrect
+- would choose to complete and return the CSV file rather than use the Google Sheets version
+
+## Future considerations
+
+We are confident that users would be able to complete the spreadsheet we tested. 
+
+Before using it, we’d need to give more thought to: 
+
+- how users could tell us about inaccuracies
+- what would happen if a user changed information in the spreadsheet or added extra SDs
+- the reference number we’d use in the spreadsheet
+- whether to offer a Google Sheets version
+- how to be clearer in the email about which courses would become available and when
+
+After the research we decided to focus on the next stage of the process. We’ll need a more streamlined way to set up users once we've gathered contact details from HEIs.
```

Comment:
Do we want to highlight what the next stage is; uploading the data via the support for apply service?

Reply:
I was keeping it brief since that will be covered in another design history entry, but I'll add a bit more detail.